main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
Are commas easier than spaces? How are commas going to be escaped if you need it in a value of a config?

I generally shy away from having multiple ways to do the same thing if they're approximately equivalent, introduces ambiguity in documentation and maintaining support for both tends to diverge.